debian/rules

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/make -f
+
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+
+%:
+	dh $@
```

Comment:
Does this invoke autotools? Because nyan should be built and packaged with cmake?